build.sbt

```diff
@@ -200,7 +200,18 @@ lazy val commonSettings = clearSourceAndResourceDirectories ++ publishSettings +
   },
   // Remove auto-generated manifest attributes
   packageOptions in Compile in packageBin := Seq.empty,
-  packageOptions in Compile in packageSrc := Seq.empty
+  packageOptions in Compile in packageSrc := Seq.empty,
+
+  // Lets us CTRL-C partest without exiting SBT entirely
+  cancelable in Global := true,
```

Comment:
You may want to reference https://github.com/sbt/sbt/issues/1442 here. Seeing the brute force `in Global` in a subproject definition made me wonder why you need it, leading to said sbt ticket.